birthdays-project/BirthdayApp/views.py

```diff
@@ -0,0 +1,21 @@
+from django.shortcuts import render
+from django.http import Http404
+
+from .models import User
+
+
+def index(request):
+    userDB = User.objects.all()
```

Comment:
'User.objects.all()' is not a 'database' but a collection of all users. So a better name would be just `users`.